### PR TITLE
docs: add new line

### PR DESCRIPTION
**Pull request by luisan00**
### Contribution description
Add a new line

### Testing procedure
See `README.md` in root folder

### PRs/Issues
None

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+# titulo
+## seccion
+### sub-seccion
+
+- lista1
+- lista2
+
```
Comment (eduazocar) [inline CONTRIBUTING.md:8]: hay un espacio vacio

Comment (luisan00) [reply CONTRIBUTING.md:8]: extra space do you mean?

Comment (eduazocar) [reply CONTRIBUTING.md:8]: yes, sorry for speak in espanish

**Review by eduazocar — APPROVED** [review]
This is a great work

**Pull request merged**